### PR TITLE
chore: remove lemon tab params

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Because we no longer require the `inline` prop thanks to https://github.com/PostHog/posthog/pull/19475 we can just remove it

## Changes

- Remove `inline` prop from `LemonTabs`
- Replace `LemonTabs` with `LemonSegmentedButtons`

## How did you test this code?

Made sure it wasn't in use anywhere

### CODE REVIEW

File: frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.tsx [hunk 1]
```diff
@@ -73,7 +71,6 @@ export function LemonTabs<T extends string | number>({
             className={clsx(
                 'LemonTabs',
                 transitioning && 'LemonTabs--transitioning',
-                inline && 'LemonTabs--inline',
                 borderless && 'LemonTabs--borderless'
```
Comment (benjackwhite) [inline frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.tsx:74]: We also don't need borderless right?

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.tsx (hunk 1 above)
Comment (daibhin) [reply frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.tsx:74]: We don't but @Gilbert09 started using it somewhere else 🙈 I was going to follow up with him and find an alternative once he's back

**Comment by daibhin** [comment]
@benjackwhite looks like the `borderless` param was removed elsewhere. This should be good enough to remove `inline` now too

**Review by benjackwhite — APPROVED** [review]

**Comment by benjackwhite** [comment]
I didnt check but i feel like this might make the scrolling look crap tbh (but i cant remember) as there is a little margin...

**Comment by daibhin** [comment]
It actually only affects this margin and not the scrolling which is a worthwhile tradeoff I think
Screenshot 2024-01-18 at 13 42 48

**Pull request merged**